web/themes/contrib/civictheme/assets/sass/layout_builder/_layout_builder.scss

```diff
@@ -0,0 +1,62 @@
+//
+// Drupal layout builder.
+//
+.path-admin {
+  .block-system-main-block {
+    #layout-builder {
+      padding: 1.5em 1.5em 0.5em;
+      border: 3px solid #2f91da;
+      background-color: #fff;
+
+      div[data-layout-builder-highlight-id^='section-'] {
+        width: 100%;
+        margin-bottom: 1.5em;
+        padding: 1.5em 0;
+        text-align: center;
+        outline: 2px dashed #979797;
+        background-color: #f7f7f7;
+      }
+
+      div[role='group'] {
+        margin-bottom: 1.5em;
+
+        main[data-layout-builder-highlight-id^='section-update-'] {
+          .ct-layout__inner.container {
+            section[class^='ct-layout__'],
+            aside[class^='ct-layout__'] {
+              outline: 2px dashed #2f91da;
+
+              div[aria-hidden] {
+                display: none;
+              }
+
+              .block.block-layout-builder {
+                padding: 1.5em;
+                cursor: move;
+                background-color: #fff;
+                position: relative;
+
+                div.contextual {
+                  display: contents;
+
+                  ul {
+                    border: 1px solid;
```

Comment:
Menu styles directly the contexual hide show. missing required classes at each level of form. Hence trigger not working 
Styles to show by default as a block